joint_trajectory_controller/doc/parameters.rst

```diff
@@ -62,11 +62,27 @@ open_loop_control (boolean)
 
   Default: false
 
+start_with_holding (bool)
+  If true, start with holding position after activation. Otherwise, no command will be sent until
```

Comment:
Rephrasing of this sentence would be helpful I felt. Two sentence seems to contradict each other. Mainly the second one sounds in a way that, JTC will not send any trajectory commands if false is set. 

Note: I'm not a native speaker, it's just that I perceived it in that way.

Reply:
But that's the way it is: if `start_with_holding` is false, no command will be sent to the hardware interfaces after startup.

Reply:
Yeah you are right, I actually did it correctly on my setup. Sorry my bad. 

However, I'm curious to know, why do we use this specific verbose for this parameter. Because, `start with holding` made me think initially like, start by not sending any commands and just hold the robot in place.

Reply:
"not sending any commands" (to the hardware) and "just hold the robot in place" do not fit well together ;)

Please leave a comment on #816 about your opinion.

Reply:
I'll resolve this here, because it is a general issue and not related to this backport.

Reply:
yes please